api-ref-assets/config/toc.yaml

```diff
@@ -0,0 +1,267 @@
+    version: v1
+  - name: ClusterRoleBinding
+    group: rbac.authorization.k8s.io
+    version: v1
+  - name: Role
+    group: rbac.authorization.k8s.io
+    version: v1
+  - name: RoleBinding
+    group: rbac.authorization.k8s.io
+    version: v1
+- name: Policy Resources
+  chapters:
+  - name: LimitRange
+    group: ""
+    version: v1
+  - name: ResourceQuota
+    group: ""
+    version: v1
+  - name: NetworkPolicy
+    group: networking.k8s.io
+    version: v1
+  - name: PodDisruptionBudget
+    group: policy
+    version: v1
+  - name: PodSecurityPolicy
+    group: policy
+    version: v1beta1
+- name: Extend Resources
+  chapters:
+  - name: CustomResourceDefinition
+    group: apiextensions.k8s.io
+    version: v1
+    otherDefinitions:
+    - CustomResourceDefinitionSpec
+    - JSONSchemaProps
+    - CustomResourceDefinitionStatus
+    - CustomResourceDefinitionList
+  - name: MutatingWebhookConfiguration
+    group: admissionregistration.k8s.io
+    version: v1
+  - name: ValidatingWebhookConfiguration
+    group: admissionregistration.k8s.io
+    version: v1
+- name: Cluster Resources
+  chapters:
+  - name: Node
+    group: ""
+    version: v1
+  - name: Namespace
+    group: ""
+    version: v1
+  - name: Event
+    group: events.k8s.io
+    version: v1
+  - name: APIService
+    group: apiregistration.k8s.io
+    version: v1
+  - name: Lease
+    group: coordination.k8s.io
+    version: v1
+  - name: RuntimeClass
+    group: node.k8s.io
+    version: v1
+  - name: FlowSchema
+    group: flowcontrol.apiserver.k8s.io
+    version: v1beta1
+  - name: PriorityLevelConfiguration
+    group: flowcontrol.apiserver.k8s.io
+    version: v1beta1
+  - name: Binding
+    group: ""
+    version: v1
+  - name: ComponentStatus
+    group: ""
+    version: v1
+- name: Common Definitions
+  chapters:
+  - name: DeleteOptions
+    key: io.k8s.apimachinery.pkg.apis.meta.v1.DeleteOptions
+  - name: LabelSelector
+    key: io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector
+  - name: ListMeta
+    key: io.k8s.apimachinery.pkg.apis.meta.v1.ListMeta
+  - name: LocalObjectReference
+    key: io.k8s.api.core.v1.LocalObjectReference
+  - name: NodeSelectorRequirement
+    key: io.k8s.api.core.v1.NodeSelectorRequirement
+  - name: ObjectFieldSelector
+    key: io.k8s.api.core.v1.ObjectFieldSelector
+  - name: ObjectMeta
+    key: io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
+  - name: ObjectReference
+    key: io.k8s.api.core.v1.ObjectReference
+  - name: Patch
+    key: io.k8s.apimachinery.pkg.apis.meta.v1.Patch
+  - name: Quantity
+    key: "io.k8s.apimachinery.pkg.api.resource.Quantity"
+  - name: ResourceFieldSelector
+    key: io.k8s.api.core.v1.ResourceFieldSelector
+  - name: Status
+    key: io.k8s.apimachinery.pkg.apis.meta.v1.Status
+  - name: TypedLocalObjectReference
+    key: io.k8s.api.core.v1.TypedLocalObjectReference
+skippedResources:
+- APIGroup
+- APIGroupList
+- APIResourceList
+- APIVersions
+- Eviction
+- Scale
```

Comment:
Not a v1.22 thing but it does make me wonder: why do we skip eviction and scale subresources?